solved issues page -> expanding issues

patchzyy opened the pull request.

patchzyy committed:
init

patchzyy committed:
expanding issues op de solved issue page

patchzyy committed:
auth update

ZoutigeWolf reviewed (commented):
Wrm delete je de SERVER_URL const

Ik kijk thuis verder

luuk180 reviewed (dismissed):
Kan je de changes aan auth.ts er uit halen en de merge conflict fixen?

ZoutigeWolf committed:
Merge branch 'master' into MyIssuesSolvedIssuesPages

ZoutigeWolf committed:
me ballen auth

ZoutigeWolf committed:
Merge remote-tracking branch 'origin/MyIssuesSolvedIssuesPages' into MyIssuesSolvedIssuesPages

# Conflicts:
#	src/api/auth.ts

ZoutigeWolf commented:
is die expanding button niet gwn kk useless, in de praktijk kunnen mensen regels en regels aan tekst schrijven in die issue en dat gaat er dan vrij raar uit zien

en de css ziet er nog steeds shit uit

luuk180 commented:
haha jij laat er geen gras over groeien he Guus

ZoutigeWolf commented:
<img width="1378" alt="image" src="https://github.com/INF2A-Team-2/VisconSupportApp/assets/80846801/eac495d4-f7e6-45e5-a2f5-d9838b3222d1">

<img width="1374" alt="image" src="https://github.com/INF2A-Team-2/VisconSupportApp/assets/80846801/99551cb5-8bb6-4085-b464-5d7e6c7106f5">

het ziet er wel beter uit maar als de tekst te lang is zie je de button niet eens
en waarom is de font weight van de button anders dan de rest
en iets meer gap tussen de tekst stukken maakt t ook wat duidelijker

patchzyy committed:
Code cleanup + dynamic text length

Code cleanup, nieuwe file aangemaakt om de dropdown menu te renderen.
ook dynamic text lengte om het minder lelijk te maken

patchzyy committed:
CSS merge

Solved and my issues gebruiken nu dezelfde css

patchzyy committed:
kleine text + meer ruimte

ZoutigeWolf committed:
move RenderIssueDetails to Components

ZoutigeWolf committed:
pici padding

ZoutigeWolf reviewed (approved):
ok
